README.md

```diff
@@ -97,6 +97,8 @@ Options:
           # The log level
 
           # Default: debug
+      [--fail_if_no_pacts_found]
```

Comment:
Thor translates the underscores into dashes, so the option is actually `--fail-if-no-pacts-found`